File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/ORCATransferOrdersDiseaseInfo.java
package nta.med.data.model.ihis.nuro;

import java.util.Date;

public class ORCATransferOrdersDiseaseInfo {
	private String diagnosisCode       ;
	private String diagnosisSystem     ;
	private Date diagnosisStartDate    ;
	private Date diagnosisEndDate      ;
	private String mmlTableId          ;
	private String diagnosisCategory   ;
	private String jusangYn            ;
	public ORCATransferOrdersDiseaseInfo(String diagnosisCode,
			String diagnosisSystem, Date diagnosisStartDate,
			Date diagnosisEndDate, String mmlTableId, String diagnosisCategory,
			String jusangYn) {
		super();
		this.diagnosisCode = diagnosisCode;
		this.diagnosisSystem = diagnosisSystem;
		this.diagnosisStartDate = diagnosisStartDate;
		this.diagnosisEndDate = diagnosisEndDate;
		this.mmlTableId = mmlTableId;
		this.diagnosisCategory = diagnosisCategory;
		this.jusangYn = jusangYn;
	}
	public String getDiagnosisCode() {
		return diagnosisCode;
	}
	public void setDiagnosisCode(String diagnosisCode) {
		this.diagnosisCode = diagnosisCode;
	}
	public String getDiagnosisSystem() {
		return diagnosisSystem;
	}
	public void setDiagnosisSystem(String diagnosisSystem) {
		this.diagnosisSystem = diagnosisSystem;
	}
	public Date getDiagnosisStartDate() {
		return diagnosisStartDate;
	}
	public void setDiagnosisStartDate(Date diagnosisStartDate) {
		this.diagnosisStartDate = diagnosisStartDate;
	}
	public Date getDiagnosisEndDate() {
		return diagnosisEndDate;
	}
	public void setDiagnosisEndDate(Date diagnosisEndDate) {
		this.diagnosisEndDate = diagnosisEndDate;
	}
	public String getMmlTableId() {
		return mmlTableId;
	}
	public void setMmlTableId(String mmlTableId) {
		this.mmlTableId = mmlTableId;
	}
	public String getDiagnosisCategory() {
		return diagnosisCategory;
	}
	public void setDiagnosisCategory(String diagnosisCategory) {
		this.diagnosisCategory = diagnosisCategory;
	}
	public String getJusangYn() {
		return jusangYn;
	}
	public void setJusangYn(String jusangYn) {
		this.jusangYn = jusangYn;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Orca/ORCATransferOrdersDiseaseInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Orca
{
    [Serializable]
    public class ORCATransferOrdersDiseaseInfo
    {
        private String _diagnosisCode;
        private String _diagnosisSystem;
        private String _diagnosisStartDate;
        private String _diagnosisEndDate;
        private String _mmlTableId;
        private String _diagnosisCategory;
        private String _jusangYn;

        public String DiagnosisCode
        {
            get { return this._diagnosisCode; }
            set { this._diagnosisCode = value; }
        }

        public String DiagnosisSystem
        {
            get { return this._diagnosisSystem; }
            set { this._diagnosisSystem = value; }
        }

        public String DiagnosisStartDate
        {
            get { return this._diagnosisStartDate; }
            set { this._diagnosisStartDate = value; }
        }

        public String DiagnosisEndDate
        {
            get { return this._diagnosisEndDate; }
            set { this._diagnosisEndDate = value; }
        }

        public String MmlTableId
        {
            get { return this._mmlTableId; }
            set { this._mmlTableId = value; }
        }

        public String DiagnosisCategory
        {
            get { return this._diagnosisCategory; }
            set { this._diagnosisCategory = value; }
        }

        public String JusangYn
        {
            get { return this._jusangYn; }
            set { this._jusangYn = value; }
        }

        public ORCATransferOrdersDiseaseInfo() { }

        public ORCATransferOrdersDiseaseInfo(String diagnosisCode, String diagnosisSystem, String diagnosisStartDate, String diagnosisEndDate, String mmlTableId, String diagnosisCategory, String jusangYn)
        {
            this._diagnosisCode = diagnosisCode;
            this._diagnosisSystem = diagnosisSystem;
            this._diagnosisStartDate = diagnosisStartDate;
            this._diagnosisEndDate = diagnosisEndDate;
            this._mmlTableId = mmlTableId;
            this._diagnosisCategory = diagnosisCategory;
            this._jusangYn = jusangYn;
        }

    }
}